src/jshint.js

```diff
@@ -2959,7 +2762,7 @@ var JSHINT = (function() {
       "(metrics)"   : null,
       "(statement)" : null,
       "(context)"   : null,
-      "(blockscope)": null,
+      "(scope)": null,
```

Comment:
I'm no fan of the object literal formatting style, either, but consistency trumps personal preference. Would you mind observing the whitespace convention here?